test/test_cuda.py

```diff
@@ -1130,7 +1130,7 @@ def _test_broadcast_coalesced(self, tensors, buffer_size):
         for old_version, (_, t) in zip(versions, bc_tensors_t):
             self.assertEqual(t._version, old_version)
             t.zero_()
-            self.assertEqual(t._version, old_version + 1)
+            self.assertGreater(t._version, old_version)
```

Comment:
What happened here?

Reply:
After moving version bumping to non-Variable dispatch functions, there is no guarantee that the version will be bumped only once in an in-place operation. In this case, `t.zero_()` bumps `t`'s version twice, first in:
```cpp
Tensor & TypeDefault::zero_(Tensor & self) const {
    const OptionalDeviceGuard device_guard(device_of(self));
    if (at::VersionUpdateMode::is_enabled()) self.bump_version();  // NOTE: bumps version
    // self.bump_version();
    return at::native::zero_(/* native_actuals */ self);
}
```
and then in:
```cpp
Tensor & CPUFloatType::_th_zero_(Tensor & self) const {
    // DeviceGuard omitted
    if (at::VersionUpdateMode::is_enabled()) self.bump_version();  // NOTE: bumps version
    // self.bump_version();
    auto self_ = checked_tensor_unwrap(self,"self",1, false, Backend::CPU, ScalarType::Float);
    THFloatTensor_zero(self_);
    return self;
}
```

Reply:
I believe we assume that TH functions won't be called directly, but only in other functions that dispatch them.  Does this mean we can avoid the bumps in TH functions?